### PR TITLE
clean up and fix styling issues

### PR DESCRIPTION
**Pull request by erickang21**
- removed a lot of divs that weren't necessary, updated the styling of other divs to cleanup code
- fixed some styling issues with misplaced items

### CODE REVIEW

File: components/practice/dropdown-blanks.js [hunk 1]
```diff
@@ -352,6 +352,19 @@ class DropdownBlanksPractice extends RequesterMixin(Localizer(LitElement)) {
 		const templates = practiceTemplates(this.localize.bind(this));
 		const template = templates[this._templateIndex];
 
+		// Before auto-suggest, check that text is under limit
+		let textToUse; // text to use for question generation
```
[the remark this answers is not in the document]
Comment (erickang21) [reply components/practice/dropdown-blanks.js:356]: i'm fairly certain textToUse needs to be there.

[the remark this answers is not in the document]
Comment (JamesVaChao) [reply components/practice/dropdown-blanks.js:356]: What is it used for?

**Comment by erickang21** [comment]
what I did to test styling changes:
- left source text too long, banner should be appropriately sized
- clicked generate, banner should appear as normal.

what i did to test activityData cleanups:
- clicked generate, optionally fill in title and instruction fields, then click insert. go back into editor and then click edit: all information should remain there.

**Comment by JamesVaChao** [comment]
conflcits need to be resolved

File: .lmsrelease [hunk 2]
```diff
@@ -1 +1 @@
-20.23.10
+20.23.9
```
Comment (JamesVaChao) [inline .lmsrelease:1]: why

File: components/plugins/practice.js [hunk 3]
```diff
@@ -651,7 +644,7 @@ class PracticeDialog extends RequesterMixin(Localizer(LitElement)) {
 			display: block;
 			margin: 0 auto;
 		}
-		`];
```
Comment (JamesVaChao) [inline components/plugins/practice.js:654]: why

File: package-lock.json [hunk 4]
```diff
@@ -1616,9 +1616,9 @@
       "dev": true
```
Comment (JamesVaChao) [inline package-lock.json:1616]: why

File: .lmsrelease (hunk 2 above)
Comment (erickang21) [reply .lmsrelease:1]: this was what happened when i merged main into this branch. the changes just appeared

File: components/practice/dropdown-blanks.js [hunk 5]
```diff
@@ -443,22 +454,20 @@ class DropdownBlanksPractice extends RequesterMixin(Localizer(LitElement)) {
...
 				this.questionText = contentFormatted;
 
 			} else if (fieldToFill === fields.answer) {
 				const answerIndex = this.activityData.questions[0].answers.map(function(answer) { return answer.id; }).indexOf(answerId);
 				if (answerIndex > -1) {
-					this.activityData.questions[0].answers[answerIndex].answers[0].text = contentFormatted;
 					this._updateValue(`questions:0:answers:${answerIndex}:answers:0:text`, contentFormatted);
-					this._updateActivity();
+				} else {
+					console.warn(`Unable to update answer. llm suggestion answer placeholder id of '${answerId}' not found`);
```
[the remark this answers is not in the document]
Comment (erickang21) [reply components/practice/dropdown-blanks.js:465]: that also took place during the merge.

**Review by JamesVaChao — APPROVED** [review]

File: components/practice/dropdown-blanks.js [hunk 6]
```diff
@@ -425,7 +437,6 @@ class DropdownBlanksPractice extends RequesterMixin(Localizer(LitElement)) {
 			} else if (content.match(fields.answer)) {
 				if (firstAnswer) {
 					const processedQuestion = this.activityData.questions[0].text.replaceAll('(', '[').replaceAll(')', ']');
-					this.activityData.questions[0].text = processedQuestion.trim();
```
Comment (JamesVaChao) [inline components/practice/dropdown-blanks.js:428]: Please do not have these changes in the branch. Keep the branch to what its suppose to be doing (fixing styling). If you really want to remove these, create a seperate branch. But at that point you'd have to remove them from all question types to be consistent.

**Review by ianmclean2011 — APPROVED** [review]

**Review by grant-cleary — APPROVED** [review]

**Pull request merged**